src/main/java/com/blogspot/mydailyjava/weaklockfree/WeakConcurrentMap.java

```diff
@@ -58,17 +58,21 @@ public WeakConcurrentMap(boolean cleanerThread) {
      * @param classLoader The class loader to check.
      * @return {@code true} if the provided class loader can be unloaded.
      */
-    private static boolean canBeUnloaded(ClassLoader classLoader) {
-        return classLoader != ClassLoader.getSystemClassLoader()
-                && classLoader != ClassLoader.getSystemClassLoader().getParent() // ext/platfrom class loader
-                && classLoader != null; // bootstrap class loader
+    private static boolean isPersistentClassLoader(ClassLoader classLoader) {
```

Comment:
I like that name much better :)

Reply:
I try to avoid negating method returns as a general rule ;)